resources/styles/components/task/index.less

```diff
@@ -5,6 +5,8 @@
 .task {
   max-width: @tutor-task-width;
   margin: 0 auto;
+
+  @import '../task-step/step-footer-mixin';
```

Comment:
Rather than nesting the imports can you make an explicit mixin in `task-footer-mixin.less` (maybe named `.task-footer-mixin()`)?